### PR TITLE
Add one case for Windows cpu model flag

### PR DESCRIPTION
**Pull request by yanglei-rh**
ID:1847466

Signed-off-by: Lei Yang <leiyang@redhat.com>

### CODE REVIEW

**Comment by yanglei-rh** [comment]
@PaulYuuu @kimi1978wy Please help me review it,thanks.

**Comment by kimi1978wy** [comment]
I cannot pass this case, it cause core dump, so it is a pre-auto case for a product bug?

File: qemu/tests/boot_e1000e_with_cpu_flag.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+from virttest import utils_net
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Boot a winodws guest add Vendor ID with name "KVMKVMKVM" to cpu model flag
+
+    1) Boot a VM with (with Q35 + e1000e + hv_vendor_id=KVMKVMKVM)
+    2) Run the bcdedit command as administrator
```
[the remark this answers is not in the document]
Comment (yanglei-rh) [reply qemu/tests/boot_e1000e_with_cpu_flag.py:11]: Thanks for your reminder, I have updated.

**Comment by kimi1978wy** [comment]
LGTM 
Acked-by: Yu Wang

 (1/2) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.e1000e.Guest.Win10.i386.io-github-autotest-qemu.boot_e1000e_with_cpu_flag.q35: PASS (328.41 s)
 (2/2) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.e1000e.Guest.Win10.x86_64.io-github-autotest-qemu.boot_e1000e_with_cpu_flag.q35: PASS (282.33 s)

File: qemu/tests/cfg/boot_e1000e_with_cpu_flag.cfg [hunk 2]
```diff
@@ -0,0 +1,8 @@
+- boot_e1000e_with_cpu_flag:
+    only q35
+    only e1000e
+    only Windows
+    type = boot_e1000e_with_cpu_flag
+    cpu_model_flags += "hv_vendor_id=KVMKVMKVM"
```
Comment (PaulYuuu) [inline qemu/tests/cfg/boot_e1000e_with_cpu_flag.cfg:6]: Missing a comma before `hv_vendor_id`.

Comment (yanglei-rh) [reply qemu/tests/cfg/boot_e1000e_with_cpu_flag.cfg]: I have add it, please help me review it again,thanks.

File: qemu/tests/boot_e1000e_with_cpu_flag.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    session_serial = vm.wait_for_serial_login(timeout=login_timeout)
+    bcdedit_debug = params["bcdedit_debug"]
+    bcdedit_cmd = params["bcdedit_cmd"]
+    guest_ip = vm.get_address()
+
+    try:
+        session_serial.cmd(bcdedit_debug)
+        session_serial.cmd(bcdedit_cmd)
+        vm.reboot(timeout=login_timeout)
+        status, output = utils_net.ping(dest=guest_ip, count=10,
```
Comment (PaulYuuu) [inline qemu/tests/boot_e1000e_with_cpu_flag.py:31]: It does not match to "Ping external host from guest."

Comment (yanglei-rh) [reply qemu/tests/boot_e1000e_with_cpu_flag.py]: Sorry, it's my mistake, I have changed it.

**Comment by yanglei-rh** [comment]
@PaulYuuu Please help me review it again,thanks.

**Comment by PaulYuuu** [comment]
LGTM.

**Pull request merged**